setup.py

```diff
@@ -134,6 +134,9 @@ def make_pybind11_extension(
     rpathflag = '-Wl,-rpath,%s/lib' % sys.exec_prefix
     if "CONDA_PREFIX" in os.environ:
         include_dirs.append(os.path.join(os.environ["CONDA_PREFIX"], "include"))
+    else:
+        include_dirs.append(os.path.join("/usr/include/scotch"))
+        include_dirs.append(os.path.join("/usr/include/metis"))
```

Comment:
Thanks for the explanation. The path ``/usr/include/scotch`` and ``/usr/include/metis`` are maintained by the OS. Maybe my test environment of the OS (Ubuntu 16.04) uses a too old version of SCOTCH that does not has metis.h, or the packing way of my OS packs SCOTCH and METIS respectively. I will check this and see what is the best way to include this path.